compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -1545,7 +1553,7 @@ object Scanners {
 
    /* Initialization: read first char, then first token */
     nextChar()
-    nextToken()
+    (this: @unchecked).nextToken()
```

Comment:
The initialization of `ToIndentParser` is unsafe because the ctor of `Parser` calls `in.nextToken()` where `in` is a `ToIndentScanner` and can access uninitialized fields in `ToIndentParser`. In practice the `if token != EMPTY` in `ToIndentScanner.nextToken` prevents this from happening.